index.d.ts

```diff
@@ -593,6 +598,17 @@ declare namespace tracer {
            */
           enabled?: boolean
         }
+      },
+
+      aiguard?: {
```

Comment:
why are these options in experimental and others out of experimental?
If these things are to configure aiguard, I think that we could add these in main `aiguard` config options, out of experimental, when we want to remove things from experimental is always a pain because it is a breaking change.

Reply:
Moved to `aiguard`, they were in experimental as they might not be in the final version. Thanks for the feedback.

Reply:
Moved everything into experimental, we can alias it later. Thanks for the input!